File: tornado/server-01-hello.py
#!/usr/bin/env python
import tornado.ioloop
import tornado.web

PORT = 8080

class MainHandler(tornado.web.RequestHandler):
    def get(self):
        self.write("Hello world")

application = tornado.web.Application([
    (r"/", MainHandler),
])

if __name__ == "__main__":
    application.listen(PORT)
    print "Listening on http://localhost:%s" % PORT
    tornado.ioloop.IOLoop.instance().start()

File: nodejs/server-01-hello.js
var http = require("http");

var PORT = 8080

http.createServer(function(request, response) {
  response.writeHead(200, {"Content-Type": "text/plain"});
  response.write("Hello World");
  response.end();
}).listen(PORT);

console.log("Listening on http://localhost:" + PORT);
